_sub/compute/k8s-kiam/vars.tf

```diff
@@ -89,6 +89,11 @@ variable "chart_version" {
   type        = string
   description = "KIAM helm chart version"
   default     = null
+
+  validation {
+    condition     = tonumber(substr(var.chart_version, 0, 1)) >= 6
```

Comment:
`condition = element(split(".", var.chart_version), 0) >= 6` should take future proof it

Reply:
Question though.  If you use a split command on a string then do the returned elements come back as strings or integers?  Is there not a need to cast data types here?

Reply:
It will be implicitly converted, I think. I verified with this code:

```hcl
locals {
  version = "6.1.2"
  first = element(split(".", local.version), 0)
}

output "ge-6" {
  value = local.first >= 6
}

output "ge-7" {
  value = local.first >= 7
}
```

Reply:
Implemented as:

`tonumber(element(split(".", var.chart_version), 0)) >= 6`